src/Jackett.Common/ExceptionWithConfigData.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Files with a change like this on line 1 are the ones that had their encoding changed to utf-8.

Several files have only this change.